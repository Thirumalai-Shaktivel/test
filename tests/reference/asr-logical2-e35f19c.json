{
    "basename": "asr-logical2-e35f19c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/logical2.f90",
    "infile_hash": "7efb2fb2352ed7c8d81b56547300d3653a2f796950815393a0fae0ab",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-logical2-e35f19c.stdout",
    "stdout_hash": "bda54518073ef9848e9375b610b7dca0fecbe6546c868950e9e1697c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}