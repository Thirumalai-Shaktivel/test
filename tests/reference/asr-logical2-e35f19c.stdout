(TranslationUnit (SymbolTable 1 {logical2: (Program (SymbolTable 2 {a: (Variable 2 a Local () () Default (Logical 4 []) Source Public Required .false.), b: (Variable 2 b Local () () Default (Logical 4 []) Source Public Required .false.)}) logical2 [] [(= (Var 2 a) (ConstantLogical .true. (Logical 4 [])) ()) (= (Var 2 b) (ConstantLogical .false. (Logical 4 [])) ()) (If (BoolOp (Var 2 a) And (Var 2 b) (Logical 4 []) ()) [(Print () [(ConstantString "Line 1 - Condition is true" (Character 1 26 () []))])] [(Print () [(ConstantString "Line 1 - Condition is false" (Character 1 27 () []))])])])}) [])
